includes/admin/discounts/edit-discount.php

```diff
@@ -127,186 +126,162 @@
-                        <label for="edd-max-uses"><?php _e( 'Max Uses', 'easy-digital-downloads' ); ?></label>
-                    </th>
-                    <td>
-                        <input type="text" id="edd-max-uses" name="max_uses" value="<?php echo esc_attr( $discount->max_uses ); ?>" placeholder="<?php esc_html_e( 'Unlimited', 'easy-digital-downloads' ); ?>" />
-                        <p class="description"><?php _e( 'The maximum number of times this discount can be used.', 'easy-digital-downloads' ); ?></p>
-                    </td>
-                </tr>
-
-                <?php do_action( 'edd_edit_discount_form_before_use_once', $discount->id, $discount ); ?>
-
-                <tr>
-                    <th scope="row" valign="top">
-                        <label for="edd-use-once"><?php _e( 'Use Once Per Customer', 'easy-digital-downloads' ); ?></label>
-                    </th>
-                    <td>
-                        <input type="checkbox" id="edd-use-once" name="once_per_customer" value="1"<?php checked( true, $single_use ); ?>/>
-                        <span class="description"><?php _e( 'Prevent customers from using this discount more than once.', 'easy-digital-downloads' ); ?></span>
-                    </td>
-                </tr>
-
-                <?php do_action( 'edd_edit_discount_form_before_status', $discount->id, $discount ); ?>
-
-                <tr>
-                    <th scope="row" valign="top">
-                        <label for="edd-status"><?php _e( 'Status', 'easy-digital-downloads' ); ?></label>
-                    </th>
-                    <td>
-                        <select name="status" id="edd-status">
-                            <option value="active" <?php selected( $discount->status, 'active' ); ?>><?php _e( 'Active', 'easy-digital-downloads' ); ?></option>
-                            <option value="inactive"<?php selected( $discount->status, 'inactive' ); ?>><?php _e( 'Inactive', 'easy-digital-downloads' ); ?></option>
-                        </select>
-                        <p class="description"><?php _e( 'The status of this discount code.', 'easy-digital-downloads' ); ?></p>
-                    </td>
-                </tr>
-
-                <?php do_action( 'edd_edit_discount_form_before_notes', $discount->id, $discount ); ?>
-
-                <tr>
-                    <th scope="row" valign="top">
-                        <label for="notes"><?php _e( 'Discount Notes', 'easy-digital-downloads' ); ?></label>
-                    </th>
-                    <td>
+
+						<label class="screen-reader-text" for="end-date-minute">
+							<?php esc_html_e( 'Expiration Date Minute', 'easy-digital-downloads' ); ?>
+						</label>
+						<input type="number" min="0" max="59" step="1" name="end_date_minute" id="end-date-minute" value="<?php echo esc_attr( false !== $discount->end_date ? $end_minute : '' ); ?>" placeholder="00" />
```

Comment:
```suggestion
						<input type="number" min="0" max="59" step="1" name="end_date_minute" id="end-date-minute" value="<?php echo esc_attr( false !== $discount->end_date ? $end_minute : '' ); ?>" placeholder="59" />
```

To match the same placeholder value as on the `Add New Discount` page.